src/main/java/core/basesyntax/service/impl/FruitTransactionCreationServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionCreationService;
+import java.util.List;
+
+public class FruitTransactionCreationServiceImpl implements FruitTransactionCreationService {
+    private static final int FIRST_POSITION = 0;
+    private static final int SECOND_POSITION = 1;
+    private static final int THIRD_POSITION = 2;
```

Comment:
it's not informative naming for the constants, it's clear from the value itself, let's call them FRUIT_TYPE_INDEX, QUANTITY_INDEX, etc